pkgs/development/python-modules/pytask/default.nix

```diff
@@ -0,0 +1,87 @@
+{
+  lib,
+  buildPythonPackage,
+  pythonOlder,
+  fetchFromGitHub,
+  hatchling,
+  hatch-vcs,
+  attrs,
+  click,
+  click-default-group,
+  networkx,
+  optree,
+  packaging,
+  pluggy,
+  rich,
+  sqlalchemy,
+  universal-pathlib,
+  pytestCheckHook,
+  deepdiff,
+  nbmake,
+  pexpect,
+  pytest-cov,
+  pytest-xdist,
+  syrupy,
+  aiohttp,
+  git,
+}:
+buildPythonPackage rec {
+  pname = "pytask";
+  version = "0.5.1";
+  format = "pyproject";
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "pytask-dev";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-b+sS+l0Rp5bb8Dh6UBv3xHYTYKFp3dD5AuLqxB3n6Go=";
+  };
+
+  nativeBuildInputs = [
+    hatchling
+    hatch-vcs
+  ];
+
+  propagatedBuildInputs = [
+    attrs
+    click
+    click-default-group
+    networkx
+    optree
+    packaging
+    pluggy
+    rich
+    sqlalchemy
+    universal-pathlib
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    git
+    deepdiff
+    nbmake
+    pexpect
+    pytest-cov
```

Comment:
```suggestion
```
please patch it out that we need that

Reply:
Turns out this was not needed at all to build the tests. There are some other test dependencies which upstream declares in pyproject.toml but are not used (deepdiff for example) should those be removed also?

Reply:
Yes, please.